packages/contracts/test/app-registry.spec.ts

```diff
@@ -50,7 +50,9 @@ describe("AppRegistry", () => {
     provider = waffle.createMockProvider();
     wallet = (await waffle.getWallets(provider))[0];
 
-    appRegistry = await waffle.deployContract(wallet, AppRegistry);
+    appRegistry = await waffle.deployContract(wallet, AppRegistry, [], {
+      gasLimit: 6000000
```

Comment:
Why is this needed?

Reply:
the default gas limit is fixed at 4 million (it is not estimated) which is too low to deploy `AppRegistry`

Reply:
Worth noting this in a comment in the code imo